USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -318,6 +357,22 @@ private string RenderFootnotes()
             }
             return footnoteHTML.ToString();
         }
-        
+        private string RenderCrossReferences()
+        {
+            StringBuilder crossRefHTML = new StringBuilder();

```

Comment:
A minor optimization you could do is to put the output string builder inside of the if and then return it at the end of the if and return an empty string outside of the if. That way the string builder doesn't get created when it isn't needed.

Reply:
Good Idea! I'll do the same for the footnotes!